resources/js/components/PaymentInstructions.vue

```diff
@@ -11,10 +11,8 @@
               Credit/Debit Card
             </h4>
             <p class="card-text">
-              Pay with your credit/debit card online. There is a $3 surcharge to cover payment processing fees.
-                <!-- TODO this needs to be toggleable with the feature flag -->
-                Prefer to pay in-person? See an officer at the shop.<br/>
```

Comment:
Even post-COVID I don't think it's a good use of time to advertise this. Open to other opinions.